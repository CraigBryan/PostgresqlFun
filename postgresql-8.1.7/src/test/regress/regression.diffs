*** ./expected/int4.out	Wed Apr  6 21:51:40 2005
--- ./results/int4.out	Thu Nov  6 10:48:49 2014
***************
*** 144,150 ****
  (3 rows)
  
  SELECT '' AS five, i.f1, i.f1 * int2 '2' AS x FROM INT4_TBL i;
! ERROR:  integer out of range
  SELECT '' AS five, i.f1, i.f1 * int2 '2' AS x FROM INT4_TBL i
  WHERE abs(f1) < 1073741824;
   five |   f1    |    x    
--- 144,158 ----
  (3 rows)
  
  SELECT '' AS five, i.f1, i.f1 * int2 '2' AS x FROM INT4_TBL i;
!  five |     f1      |    x    
! ------+-------------+---------
!       |           0 |       0
!       |      123456 |  246912
!       |     -123456 | -246912
!       |  2147483647 |      -2
!       | -2147483647 |       2
! (5 rows)
! 
  SELECT '' AS five, i.f1, i.f1 * int2 '2' AS x FROM INT4_TBL i
  WHERE abs(f1) < 1073741824;
   five |   f1    |    x    
***************
*** 155,161 ****
  (3 rows)
  
  SELECT '' AS five, i.f1, i.f1 * int4 '2' AS x FROM INT4_TBL i;
! ERROR:  integer out of range
  SELECT '' AS five, i.f1, i.f1 * int4 '2' AS x FROM INT4_TBL i
  WHERE abs(f1) < 1073741824;
   five |   f1    |    x    
--- 163,177 ----
  (3 rows)
  
  SELECT '' AS five, i.f1, i.f1 * int4 '2' AS x FROM INT4_TBL i;
!  five |     f1      |    x    
! ------+-------------+---------
!       |           0 |       0
!       |      123456 |  246912
!       |     -123456 | -246912
!       |  2147483647 |      -2
!       | -2147483647 |       2
! (5 rows)
! 
  SELECT '' AS five, i.f1, i.f1 * int4 '2' AS x FROM INT4_TBL i
  WHERE abs(f1) < 1073741824;
   five |   f1    |    x    

======================================================================

*** ./expected/int8.out	Sun Dec  4 23:13:52 2005
--- ./results/int8.out	Thu Nov  6 10:48:49 2014
***************
*** 64,70 ****
  (5 rows)
  
  SELECT '' AS three, q1, q2, q1 * q2 AS multiply FROM INT8_TBL;
! ERROR:  bigint out of range
  SELECT '' AS three, q1, q2, q1 * q2 AS multiply FROM INT8_TBL
   WHERE q1 < 1000 or (q2 > 0 and q2 < 1000);
   three |        q1        |        q2        |      multiply      
--- 64,78 ----
  (5 rows)
  
  SELECT '' AS three, q1, q2, q1 * q2 AS multiply FROM INT8_TBL;
!  three |        q1        |        q2         |       multiply       
! -------+------------------+-------------------+----------------------
!        |              123 |               456 |                56088
!        |              123 |  4567890123456789 |   561850485185185047
!        | 4567890123456789 |               123 |   561850485185185047
!        | 4567890123456789 |  4567890123456789 |  4868582358072306617
!        | 4567890123456789 | -4567890123456789 | -4868582358072306617
! (5 rows)
! 
  SELECT '' AS three, q1, q2, q1 * q2 AS multiply FROM INT8_TBL
   WHERE q1 < 1000 or (q2 > 0 and q2 < 1000);
   three |        q1        |        q2        |      multiply      

======================================================================

