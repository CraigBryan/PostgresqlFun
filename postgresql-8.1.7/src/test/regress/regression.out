parallel group (13 tests):  char text name int2 int8 varchar float4 boolean oid int4 float8 bit numeric
     boolean              ... ok
     char                 ... ok
     name                 ... ok
     varchar              ... ok
     text                 ... ok
     int2                 ... ok
     int4                 ... FAILED
     int8                 ... FAILED
     oid                  ... ok
     float4               ... ok
     float8               ... ok
     bit                  ... ok
     numeric              ... ok
test strings              ... ok
test numerology           ... ok
parallel group (20 tests):  path time timetz point polygon comments lseg reltime interval tinterval abstime inet timestamp circle timestamptz date box type_sanity oidjoins opr_sanity
     point                ... ok
     lseg                 ... ok
     box                  ... ok
     path                 ... ok
     polygon              ... ok
     circle               ... ok
     date                 ... ok
     time                 ... ok
     timetz               ... ok
     timestamp            ... ok
     timestamptz          ... ok
     interval             ... ok
     abstime              ... ok
     reltime              ... ok
     tinterval            ... ok
     inet                 ... ok
     comments             ... ok
     oidjoins             ... ok
     type_sanity          ... ok
     opr_sanity           ... ok
test geometry             ... ok
test horology             ... ok
test insert               ... ok
test create_function_1    ... ok
test create_type          ... ok
test create_table         ... ok
test create_function_2    ... ok
test copy                 ... ok
parallel group (7 tests):  create_operator create_aggregate vacuum inherit create_misc triggers constraints
     constraints          ... ok
     triggers             ... ok
     create_misc          ... ok
     create_aggregate     ... ok
     create_operator      ... ok
     inherit              ... ok
     vacuum               ... ok
parallel group (2 tests):  create_view create_index
     create_index         ... ok
     create_view          ... ok
test sanity_check         ... ok
test errors               ... ok
test select               ... ok
parallel group (19 tests):  select_distinct_on select_distinct update hash_index btree_index union random aggregates select_into select_having namespace select_implicit case arrays join portals transactions subselect prepared_xacts
     select_into          ... ok
     select_distinct      ... ok
     select_distinct_on   ... ok
     select_implicit      ... ok
     select_having        ... ok
     subselect            ... ok
     union                ... ok
     case                 ... ok
     join                 ... ok
     aggregates           ... ok
     transactions         ... ok
     random               ... ok
     portals              ... ok
     arrays               ... ok
     btree_index          ... ok
     hash_index           ... ok
     update               ... ok
     namespace            ... ok
     prepared_xacts       ... ok
test privileges           ... ok
test misc                 ... ok
parallel group (6 tests):  select_views portals_p2 dependency rules cluster foreign_key
     select_views         ... ok
     portals_p2           ... ok
     rules                ... ok
     foreign_key          ... ok
     cluster              ... ok
     dependency           ... ok
parallel group (14 tests):  limit polymorphism prepare conversion copy2 without_oid rowtypes rangefuncs temp sequence truncate domain alter_table plpgsql
     limit                ... ok
     plpgsql              ... ok
     copy2                ... ok
     temp                 ... ok
     domain               ... ok
     rangefuncs           ... ok
     prepare              ... ok
     without_oid          ... ok
     conversion           ... ok
     truncate             ... ok
     alter_table          ... ok
     sequence             ... ok
     polymorphism         ... ok
     rowtypes             ... ok
test stats                ... ok
test tablespace           ... ok
